src/main/java/org/somerville/swag/display/LandingPage.java

```diff
@@ -1,47 +1,96 @@
+        listOfProducts.setSelectedValue(allProducts.get(0), true);
+
+        listOfProducts.addListSelectionListener(actionEvent -> {
+            Product selectedProduct = listOfProducts.getSelectedValue();
+            showFrameWithProduct(selectedProduct);
+        });
+
         signUpButton.addActionListener(actionEvent -> {
-            new JFrameBuilder.Builder().buildDefaultJFrame(new SignUp(oldFrame, customer).root,true);
+            new JFrameBuilder.Builder().buildDefaultJFrame("Sign Up", new SignUp(oldFrame, customer).root,true);
             SwingUtilities.getWindowAncestor(root).dispose();
         });
 
         logInButton.addActionListener(actionEvent -> {
-            new JFrameBuilder.Builder().buildDefaultJFrame(new LogIn(oldFrame, customer).root,true);
+            new JFrameBuilder.Builder().buildDefaultJFrame("Log In", new LogIn(oldFrame, customer).root,true);
             SwingUtilities.getWindowAncestor(root).dispose();
         });
 
+        logOutButton.addActionListener(e -> customer.logOut());
+
         viewBasketButton.addActionListener(actionEvent -> {
-            new JFrameBuilder.Builder().buildDefaultJFrame(new Basket(oldFrame, customer).root,true);
+            new JFrameBuilder.Builder().buildDefaultJFrame("Your Basket", new Basket(oldFrame, customer).root,true);
             SwingUtilities.getWindowAncestor(root).dispose();
+
         });
 
         addToBasketButton.addActionListener(actionEvent -> customer.addProductToBasket());
     }
 
+    private HashMap<Integer, Product> getAllProducts() {
+        Product productA = new Product(0,"Product A","Description C", BigDecimal.valueOf(19.99), 5, "src/main/resources/product.images/derek_crocs.png");
+        Product productB = new Product(0,"Product B","Description B", BigDecimal.valueOf(29.99), 10, "src/main/resources/product.images/derek_snowglobe.png");
+        Product productC = new Product(0,"Product C","Description B", BigDecimal.valueOf(29.99), 10, "src/main/resources/product.images/derek_snowglobe.png");
+
+        HashMap<Integer, Product> productMap = new HashMap<>(); //set this to the DB read function
+        productMap.put(0, productA);
+        productMap.put(1, productB);
+        productMap.put(2, productC);
+        return productMap;
+    }
+
+    private void showFrameWithProduct(Product product) {
+        SpinnerModel model = new SpinnerNumberModel(1, 1, product.getStockLevel(), 1);
+        JSpinner spinner = new JSpinner();
+        JFormattedTextField spin = ((JSpinner.DefaultEditor) spinner.getEditor()).getTextField();
+        spin.setEditable(false);
+        quantitySpinner.setModel(model);
+
+        lblDesc.setText(product.getDescription());
+        lblProductTitle.setText(product.getName());
+
+        ImageIcon productImageIcon1 = new ImageIcon(product.getImagepath());
+        Image scaledProductImage1 = productImageIcon1.getImage().getScaledInstance(400, 350, Image.SCALE_SMOOTH);
```

Comment:
Having set size for the images with smooth scaling is a good idea, as they are different sizes